packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -0,0 +1,193 @@
+    extends ThemedComponentBaseProps<{
+        menu: typeof Menu;
+        menuItem: typeof MenuItem;
+        button: typeof Button;
+        group: typeof CrudMoreActionsGroup;
+        divider: typeof CrudMoreActionsDivider;
+        chip: typeof Chip;
+    }> {
+    selectionSize?: number;
+    overallActions?: Maybe<ActionItem>[];
+    selectiveActions?: Maybe<ActionItem>[];
+}
+
+interface CrudMoreActionsGroupProps {
+    groupTitle: React.ReactNode;
+    menuListProps?: MenuListProps;
+    typographyProps?: React.ComponentProps<typeof Typography>;
+}
+
+function CrudMoreActionsGroup({ groupTitle, children, menuListProps, typographyProps }: PropsWithChildren<CrudMoreActionsGroupProps>) {
+    return (
+        <>
+            <Typography variant="overline" color={(theme) => theme.palette.grey[500]} sx={{ padding: "20px 15px 0 15px" }} {...typographyProps}>
+                {groupTitle}
+            </Typography>
+            <MenuList {...menuListProps}>{children}</MenuList>
+        </>
+    );
+}
+
+const CrudMoreActionsDivider = createComponentSlot(Divider)<CrudMoreActionsMenuClassKey>({
+    componentName: "CrudMoreActionsDivider",
+    slotName: "divider",
+})(
+    ({ theme }) => css`
+        margin: 8px 10px;
+        border-color: ${theme.palette.grey[50]};
+    `,
+);
+
+const MoreActionsSelectedItemsChip = createComponentSlot(Chip)<CrudMoreActionsMenuClassKey>({
+    componentName: "MoreActionsSelectedItemsChip",
+    slotName: "chip",
+})(
+    css`
+        width: 20px;
+        height: 20px;
+        flex-shrink: 0;
+        border-radius: 20px;
+        margin-left: 1px;
+    `,
+);
+
+const MoreActionsButton = createComponentSlot(Button)<CrudMoreActionsMenuClassKey>({
+    componentName: "MoreActionsButton",
+    slotName: "button",
+})(
+    css`
+        margin: 0 10px;
+    `,
+);
+
+const MoreActionsMenuItem = createComponentSlot(MenuItem)<CrudMoreActionsMenuClassKey>({
+    componentName: "MoreActionsMenuItem",
+    slotName: "menuItem",
+})(
+    css`
+        padding: 8px 15px 8px 30px !important;
+        column-gap: 10px;
+    `,
+);
+
+export function CrudMoreActionsMenu({ slotProps, overallActions, selectiveActions, selectionSize }: CrudMoreActionsMenuProps) {
+    const {
+        menu: menuProps,
+        button: buttonProps,
+        group: groupProps,
+        divider: dividerProps,
+        chip: chipProps,
+    } = slotProps ?? { menu: {}, button: {}, group: {}, divider: {}, chip: {} };
+
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => setAnchorEl(event.currentTarget);
+
+    const handleClose = () => setAnchorEl(null);
+
+    return (
+        <>
+            <MoreActionsButton variant="text" color="inherit" endIcon={<MoreVertical />} {...buttonProps} onClick={handleClick}>
+                <FormattedMessage id="comet.pages.dam.moreActions" defaultMessage="More actions" />
```

Comment:
All the message IDs probably shouldn't include `pages` or `dam` here, since this does not explicitly have anything to do with those things.

Reply:
Thanks for the hint 👍🏻

Reply:
https://github.com/vivid-planet/comet/pull/2115/commits/f0ffceb43b0dc0247b6128e5bf2d67244ec53b63